### PR TITLE
Skip evaluation for unset passmarks (required by Edip/Sdip)

### PR DESCRIPTION
**Pull request by lukastymo**

### CODE REVIEW

File: app/services/onlinetesting/phase1/Phase1TestEvaluation.scala [hunk 1]
```diff
@@ -18,23 +18,29 @@ package services.onlinetesting.phase1
...
-import model.exchange.passmarksettings.{ PassMarkThreshold, Phase1PassMarkSettings }
+import model.exchange.passmarksettings.{ PassMarkThreshold, Phase1PassMarkSettings, Phase1PassMarkThresholds }
 import model.persisted.{ SchemeEvaluationResult, TestResult }
 
 trait Phase1TestEvaluation {
 
   def evaluateForGis(schemes: List[SchemeType], sjqTestResult: TestResult, passmark: Phase1PassMarkSettings): List[SchemeEvaluationResult] = {
-    schemes map { scheme =>
-      val schemePassmark = passmark.findPassmarkForScheme(scheme)
+    for {
```
Comment (satishkamatam) [inline app/services/onlinetesting/phase1/Phase1TestEvaluation.scala]: shall we refactor this to extract into a method to be reused with gis and non gis

File: app/services/onlinetesting/phase1/Phase1TestEvaluation.scala [hunk 2]
```diff
@@ -46,7 +52,7 @@ trait Phase1TestEvaluation {
         case (Green, Green) => Green
       }
 
-      SchemeEvaluationResult(scheme, result.toString)
+      SchemeEvaluationResult(schemeToEvaluate, result.toString)
```
Comment (satishkamatam) [inline app/services/onlinetesting/phase1/Phase1TestEvaluation.scala]: Though the change is not part of this PR ... any reason why we convert to result.toString, instead of using result which is type safe ??

Comment (lukastymo) [inline app/services/onlinetesting/phase1/Phase1TestEvaluation.scala]: I think this is inherited from fasttrack. Agree, It should be type safe

File: app/services/onlinetesting/phase1/Phase1TestEvaluation.scala (hunk 1 above)
Comment (lukastymo) [inline app/services/onlinetesting/phase1/Phase1TestEvaluation.scala]: I was thinking about this. Now you assured me it's a good idea
